pulumi/infra/postgres.py

```diff
@@ -104,9 +105,27 @@ def __init__(
 
         postgres_config = PostgresConfigValues.from_config()
 
+        # Quick diatribe:
+        # "The database name is the name of a database hosted in your DB instance.
+        #  A database name is not required when creating a DB instance.
+        #  Databases hosted by the same DB instance must have a unique name within
+        #  that instance."
+        # Since we're using 1 database per instance right now, I'm going to
+        # hardcode it to the default value of `postgres` (like the postgres
+        # docker image does).
+        #
+        # As for giving the instance a name we can see on the console, that's
+        # the `identifier=`.
+        database_name = "postgres"
```

Comment:
While this should be technically OK, I think it would be good to give it a unique name, like `pluginregistry`, `PluginRegistry`, etc.

Reply:
Potentially. I'm not _actually_ sure how important these "name"s will be in the long run if we just have one db per host.